Frameworks/Foundation/NSString.mm

```diff
@@ -493,12 +493,12 @@ - (const char*)UTF8String {
  @Notes
 */
 - (BOOL)getBytes:(void*)buffer
-       maxLength:(NSUInteger)maxBuf
-      usedLength:(NSUInteger*)usedLength
-        encoding:(NSStringEncoding)encoding
-         options:(NSStringEncodingConversionOptions)options
-           range:(NSRange)range
-  remainingRange:(NSRangePointer)left {
+         maxLength:(NSUInteger)maxBuf
```

Comment:
looks like our clang-format got updated (10/31) since the last change to this file (early sep)